hal/ia32/interrupts.c

```diff
@@ -97,21 +101,26 @@ unsigned int _interrupts_multilock;
 
 void _interrupts_apicACK(unsigned int n)
 {
+	if (n == TLB_IRQ) {
+		__asm__ volatile("movl $0, (0xfee000b0)" ::);
+		return;
+	}
```

Comment:
Instead inline asm

```suggestion
	volatile u32 *p = (void *)0xfee000b0;

	if (n == TLB_IRQ) {
		*p = 0;
		return;
	}
```
and in line `114`
```c
		*p = 0;
```